src/library/library.cpp

```diff
@@ -378,3 +552,248 @@ void Library::slotSetTrackTableRowHeight(int rowHeight) {
     m_iTrackTableRowHeight = rowHeight;
     emit(setTrackTableRowHeight(rowHeight));
 }
+
+void Library::slotSetHoveredFeature(LibraryFeature* pFeature) {
+    m_hoveredFeature = pFeature;
```

Comment:
Mmmm since this element has not a parent behavior in this case I don't know what to do... This ownership never deletes the pointer because all the features are deleted on the Library destruction using a `shared_ptr` may be the best solution but it implies changing many code... What would you suggest?